Competition2017/src/competition/subsystems/agitator/commands/RunEjectAgitatorIfWheelAtSpeedCommand.java

```diff
@@ -0,0 +1,26 @@
+package competition.subsystems.agitator.commands;
+
+import competition.subsystems.agitator.AgitatorSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem;
+
+public class RunEjectAgitatorIfWheelAtSpeedCommand extends BaseAgitatorCommand {
+    
+    protected final ShooterWheelSubsystem shooterWheelSubsystem;
+    
+    public RunEjectAgitatorIfWheelAtSpeedCommand(AgitatorSubsystem agitatorSubsystem, ShooterWheelSubsystem shooterWheelSubsystem) {
+        super(agitatorSubsystem);
+        this.shooterWheelSubsystem = shooterWheelSubsystem;
+    }
+
+    @Override
+    public void initialize() {
+        log.info("Initializing");
+    }
+
+    @Override
+    public void execute() {
+        if(shooterWheelSubsystem.wheelAtSpeed()){
+            agitatorSubsystem.eject();
```

Comment:
Add an `else` to set the power to zero, otherwise the motor will continue spinning.